src/http-requests.ts

```diff
@@ -17,6 +17,22 @@ class HttpRequests {
     }
     this.url = config.host
   }
+  static constructCorrectPath(pathname: string, url: string): string {
+    let slash = '/'
+    if (pathname.endsWith('/') || url.startsWith('/')) {
+      slash = ''
+    }
+    if (pathname.endsWith('//')) {
+      pathname = pathname.substring(1)
```

Comment:
Removing the *first* character if pathname *ends* with double slash is probably not intended.

Reply:
Yes, my mistake. Thanks a lot. weird that it went by the tests without noticing. I will add some more.